### ISSUE TITLE
Installation with composer running inside docker issue

### ISSUE THREAD

**Issue by trompx** [timber/timber#2219]
Hello,

I get the following error when trying to install via composer running inside docker:
```
- Installing timber/timber (1.15.2): Downloading (100%)
Failed to execute (12) unzip -qq  '/tmp/bundles/fineallure-core-1.0.0/vendor/timber/timber/034d393132ff3ef11803d4e9f9e7b833' -d '/tmp/bundles/fineallure-core-1.0.0/vendor/composer/baa75e58'
error: invalid zip file with overlapped components (possible zip bomb)
The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
Unzip with unzip command failed, falling back to ZipArchive class
```

I develop in a Ubuntu Vagrant VM and have composer running in a docker container.
After googling it appears that this issue is caused by a no caseSensitive filesystem.
I installed composer on the host machine and everything works fine so this is really a problem with docker.

Not sure which files are causing this issue, not really more info with `composer install -vvv`

## Expected behavior
Should be able to unzip

## Actual behavior
Failed to execute (12) unzip -qq

## What version of WordPress, PHP and Timber are you using?
1.15.2

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
With composer inside docker

**Comment by trompx** [comment]
Host machine (success):
- php : 5.*
- composer: 1.10.1

Docker (fails):
- php 7.2.27
- composer: 1.10.1

Thanks to that plus a hint in this [issue](https://github.com/Ocramius/PackageVersions/issues/107#issuecomment-520858565), I upgraded my docker image to alpine 3.11 thus upgrading my php version and solving the problem.

Docker (success):
- php 7.3.16
- composer: 1.10.1

**Comment by jarednova** [comment]
Thanks for the info @trompx. I'm guessing this is related to decencies given the error and it referencing the `vendor/composer` directory.  I'd be curious about whether you encounter this error with the 2.x branch? We've done a lot of work there to reduce/clean-up dependencies that hopefully solve this (and would even give us a pointer on how to fix this in 1.x)

**Comment by emanuellopes** [comment]
I had this error, I solved this updating the php to Last version 7.4

**Comment by Levdbas** [comment]
Closing as stale
